### ISSUE TITLE
Link our page for testing-farm result, not TF page directly

### ISSUE THREAD

**Issue by jpopelka** [packit/packit-service#506]
When there's no `jobs:` specified in `.packit.yaml` we default to `tests` job (with `fedora-stable` target).

It works fine, creates SRPM, builds RPMs in copr and runs tests in testing farm.
The thing is that once the copr build is done and tests start, the URL in checks `Details` doesn't lead to packit-service build logs anymore and leads to logs from testing farm.
That is inconvenient, because then I have no way how to easy get the build logs.

For example in https://github.com/packit-service-testing-organisation/source-git-rpm/pull/2 there's no way how to know that the build logs are [here](https://prod.packit.dev/copr-build/2363/logs) and [here](https://prod.packit.dev/copr-build/2364/logs).

**Comment by lachmanfrantisek** [comment]
Isn't this a "feature"?..;)

- If you don't set the build job, you don't care about the build. (It's only a detail to be able to run tests.)
- The only situation, you care about the logs is when the build fails.

**Comment by jpopelka** [comment]
Yes, that's one way how to look at it.

I can, however, imagine that when the test job fails and there's not enough info in the testing farm logs I'd like to see the logs from the copr build as well.
But I have no suggestion about how to do that so that it doesn't break the UX so I'm fine with closing this at some grooming.

**Comment by lachmanfrantisek** [comment]
>  I'm fine with closing this at some grooming.

**Comment by TomasTomecek** [comment]
I'd say we should always link to our logs and provide the link to TFT interface from there

we can also post a comment to the PR :see_no_evil:

**Comment by lachmanfrantisek** [comment]
We can discuss with Miro, if we can steal their logs during the process. (OR, at least after that.)

**Comment by lachmanfrantisek** [comment]
Let's implement the Tomáš's suggestion.

AC:
- [ ] always link to our interface
- [ ] add TF link to our interface

**Comment by csomh** [comment]
Still relevant.

**Comment by lachmanfrantisek** [comment]
We agreed that we want to reuse the build-result page and include the test results if they are present in the database.

Do we have all the information from testing-farm stored in the database?

**Comment by TomasTomecek** [comment]
> Do we have all the information from testing-farm stored in the database?

not sure if web_url is enough https://github.com/packit/packit-service/blob/master/packit_service/models.py#L1086

**Comment by lachmanfrantisek** [comment]
We probably need to enhance the scheme to contain the results.

**Comment by csomh** [comment]
We need this.

**Comment by lachmanfrantisek** [comment]
Closing in favour of #1033
